app/Http/Controllers/BlogController.php

```diff
@@ -2,47 +2,278 @@
 
 namespace App\Http\Controllers;
 
-use App\Post;
-use Illuminate\Http\Request;
+use App\Models\User;
+use App\Models\Post;
+use App\Models\Like;
+use App\Models\Comment;
 
-use App\Http\Requests;
-use Illuminate\Session\Store;
+use JWTAuth;
+use Tymon\JWTAuth\Exceptions\JWTException;
+use Validator;
+use Illuminate\Http\Request;
 
 class BlogController extends Controller
```

Comment:
Add a PHPDoc comment to every class and method declaration. At the minimum, it should contain a description, return type and parameter declarations, if any.

This ensures that your code is documented properly and it's easy to understand.